javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,197 @@
+const firstName = document.getElementById("firstName");
+const surName = document.getElementById("surName");
+const fullNameButton = document.getElementById("fullNameButton");
+const showFullName = document.getElementById("showFullName");
+
+function fullName() {
+  showFullName.innerHTML = `${firstName.value} ${surName.value}`;
+  showFullName.classList.add("show");
+  return showFullName;
+}
+
+/*
+
+function fullName(firstName,surName){
+    return `${firstName} ${surName}`;
+}
+ */
+
+// Formal FullName
+
+const firstFormalName = document.getElementById("firstFormalName");
+const surFormalName = document.getElementById("surFormalName");
+const fullFormalNameButton = document.getElementById("fullFormalNameButton");
+const showFullFormalName = document.getElementById("showFullFormalName");
+const useFormalName = document.getElementById("useFormalName");
+
+function fullFormalName(firstFormalName, surFormalName, useFormalName) {
+  const fullName = useFormalName.checked
+    ? `Lord ${firstFormalName.value} ${surFormalName.value}`
+    : `${firstFormalName.value} ${surFormalName.value}`;
```

Comment:
I like how you split this up into 3 separate lines, makes it super easy to read. 👍🏾